charts/agent/README.md

```diff
@@ -1,3 +1,6 @@
+> [!WARNING]
+> This chart is deprecated. It does not work well with versions of Kubernetes >= 1.26. Please, use the [Semaphore custom controller chart](../controller) to run Semaphore jobs in your Kubernetes cluster.
+
```

Comment:
Can you also update the root repo README file with the same depreciation notice and link to the controller chart?
It is the first place that people will look.

Reply:
Root README updated.